GoodMorning/GoodMorning/CheckBoxButton.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CheckBoxButton.swift
+//  GoodMorning
+//
+//  Created by Miro on 2023/08/04.
+//
+
+import UIKit
+
+class CheckBoxButton: UIButton {
+
+    var isChecked = false {
+        didSet {
+            self.updateCheckMarkImage()
+        }
+    }
+
+    weak var delegate: CheckBoxButtonDelegate?
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configureAttributes()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func configureAttributes() {
+        backgroundColor = .white
+        layer.cornerRadius = 10
+        layer.borderColor = UIColor.gray.cgColor
```

Comment:
ㅇ


```suggestion
layer.borderColor = UIColor.design(.checkBox)?.withAlphaComponent(0.2).cgColor
```

Figma에서 그렸던 UI의 색상을 적용해보는 건 어떨까요? 미묘한 차이긴 합니다 ㅎㅎ

Reply:
제가 생각해보니까 figma를 보면서 색상 설정을 하진 않았네요!.. 감사합니다! 🙇‍♂️

> 커밋: b2c364c